toronto/src/pages/UserProfile.jsx

```diff
@@ -61,19 +74,25 @@ const UserProfile = () => {
             <ProfileWrapper>
               <Text size={18}>{user.email}</Text>
               <Text size={18}>{user.username}</Text>
-              <StyledLink to={'edit'}>
-                <Button>프로필 편집</Button>
-              </StyledLink>
+              {loginUser ? (
+                userId === loginUser._id ? (
+                  <StyledLink to={'edit'}>
+                    <Button>프로필 편집</Button>
+                  </StyledLink>
+                ) : (
+                  <div></div>
+                )
+              ) : (
+                <div></div>
+              )}
             </ProfileWrapper>
           </ProfileSection>
           <Tab>
             <Tab.Item title='내 게시물' index='item1'>
-              {/* TODO: User 객체에 포스트를 작성 후  */}
-              {/* {user.posts ? <PostList posts={user.posts} /> : []} */}
-              {postsDummy ? <PostList posts={postsDummy} /> : []}
+              {user.posts ? <PostList posts={myPosts} /> : []}
             </Tab.Item>
             <Tab.Item title='좋아요 게시물' index='item2'>
-              <PostList posts={likesPosts} />
+              {user.likes ? <PostList posts={likesPosts} /> : []}
```

Comment:
삼항 연산자 대신 단축 평가를 사용하면 더 간단하게 나타낼 수 있을 것 같습니다! 😄 
```suggestion
              {user.likes && <PostList posts={likesPosts} />}
```

Reply:
UserProfile쪽에서 Context 테스트 때문에 merge하고 해당 내용 추후 반영해두겠습니다.

Reply:
넵 알겠습니다!